tests/golden/copy-command/config.toml

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+main_backend = "vault-local"
+
+[[backend]]
+type = "vault-kv"
+name = "vault-local"
+address = "localhost:8200"
+mount = "secret"
+token = "<vault token>"
```

Comment:
Hmm these files don't have a token, which means I have to edit all of them in order to run these tests....

I think we should hardcode the token as:

```
token = "root"
```

Then I can just start `vault` with:

```
vault server -dev -dev-root-token-id "root"
```

And `make bats` will work without having to modify any files.

Reply:
Wrote a follow-up issue here: https://github.com/serokell/coffer/issues/42

Reply:
I've changed these tokens, but have got interesting behavior. If in main `config.toml` token isn't `root`, then tests would fail. I think that `coffer` depends on main config file and on `config.toml` in the specific test directory and I don't know why.

Reply:
Done